package.json

```diff
@@ -27,6 +27,8 @@
     "jsonwebtoken": "^8.3.0",
     "mongoose": "^5.1.7",
     "multer": "^1.3.1",
-    "nodemon": "^1.18.3"
+    "ng": "0.0.0",
```

Comment:
I don't think we need the `ng` and `npm` packages. Could you please remove these @michaeljones32 ?

Reply:
Will do now, sorry!